docs/type_reference.md

```diff
@@ -19,7 +19,9 @@ they fulfill the same expectations.
 | Function (parfor.h)     | Any function; args are specified with `Args`.     |
 | FuserT                  | [`BasicGateFuser`](lib/fuser_basic.h)             |
 | Gate                    | [`Gate`](lib/gate.h)                              |
+| GateQSim                | [`GateQSim`](lib/gates_def.h)                     |
 | GateDef                 | [`GateRX`](lib/gatedef.h)                         |
+| GK / GateKind           | [`Gate`](lib/gate.h)
```

Comment:
This link should probably be [`GateKind`](lib/gate.h) to distinguish from the `Gate` type above.

Reply:
Fixed.